lib/patch-series.ts

```diff
@@ -86,20 +86,20 @@ export class PatchSeries {
             match = tagMessage.match(/^[\s\S]*?\n\n([\s\S]*)/);
             (match ? match[1] : tagMessage).split("\n").map((line) => {
                 // tslint:disable-next-line:max-line-length
-                match = line.match(/https:\/\/lore\.kernel\.org\/.*\/([^\/]+)/);
```

Comment:
Oh right, this is my fault. I am _so_ used to escaping the slash in regular expressions that I escape it even within character ranges... Thanks for cleaning up my mess!